docker/shadow-hand/kinetic/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM shadowrobot/build-tools:xenial-kinetic-ide
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/ansible/deploy.sh"
+ENV remote_additional_bashrc_cmds="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/F_new_docker_oneliner_kinetic/docker/utils/additional_bashrc_cmds_kinetic"
```

Comment:
@toliver suggested to move it to xenial-kinetic-ide level (I mean the whole installation of this bashrc script. This is good idea. Also please use this approach https://github.com/shadow-robot/sr-build-tools/blob/master/docker/ros/kinetic/base/Dockerfile#L23 to URLs